suite-native/module-settings/src/components/EjectWallets/WalletRememberModeIconButton.tsx

```diff
@@ -20,7 +16,7 @@ export const WalletRememberModeIconButton = ({ device }: { device: TrezorDevice
 
     const handleEjectWallet = () => {
         if (device.connected) {
-            dispatch(toggleRememberDevice({ device }));
+            dispatch(deviceActions.rememberDevice({ device }));
```

Comment:
Ah, here I see that this was a use-case for `toggleRemember`.
As per my comment in `deviceActions`, I'd prefer something like this, but do as per your discretion.
```suggestion
            dispatch(deviceActions.setRememberDevice({ device, rememer: !device.remember }));
```

Reply:
https://github.com/trezor/trezor-suite/pull/22716/commits/e80f60e384cdc0f76a63d06914900d81de6f3a19